packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -0,0 +1,16 @@
+/** File Previewer */
+
+import React from 'react';
+import { Button, Card } from '../../../../primitives';
+
+export function Previewer(): JSX.Element {
```

Comment:
Not blocking, but wondering if we really need `JSX.Element` because I think it's implied from the return statement. What do you think? @ErikCH @calebpollman

Reply:
Valid question @wlee221 It's a linting requirement to ensure that exports have return type signatures